repos/system_upgrade/el8toel9/actors/postgresqlcheck/libraries/postgresqlcheck.py

```diff
@@ -0,0 +1,55 @@
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+
+# Summary for postgresql-server report
+report_server_inst_summary = (
+    'PostgreSQL server component will be upgraded. Since RHEL-9 includes only'
+    ' PostgreSQL server 13 by default, which is incompatible with 9.6, 10 and 12'
+    ' included in RHEL-8, it is necessary to proceed with additional steps'
+    ' for the complete upgrade of the PostgreSQL data.'
+)
+
+report_server_inst_hint = (
+    'Back up your data before proceeding with the upgrade'
+    ' and follow steps in the documentation section "Migrating to a RHEL 9 version of PostgreSQL"'
+    ' after the upgrade.'
+)
+
+# Link URL for postgresql-server report
+report_server_inst_link_url = 'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9-beta/html/configuring_and_using_database_servers/using-postgresql#migrating-to-a-rhel-9-version-of-postgresql_using-postgresql'  # noqa: E501; pylint: disable=line-too-long
```

Comment:
I think we should rather have a KB article here, as we're more flexible in changing it and it can later serve as a land-page for postgresql updates (so it would include the actual links to migration doc and other info).

Reply:
thanks, definitely make sense